glossary.md

```diff
@@ -140,10 +158,20 @@ A genre within AVNs that focuses on building emotional connections and romantic
 
 A method of naming MC's sibling to bypass the Patreon incest restrictions. Some games will let you change the title of this roommate to retain immersion, while other games will directly search and replace "sister" for "roommate". As a result, many games will reference how taboo it is to 'have sex with your roommate', confusing those not in the know. See [Landlady](#landlady) for mother/child.
 
+## Sandbox
+
+Games that center their gameplay on daytime cycles or a world map that the player can access to engage in a world event. There are story arcs that are only accessible at different points in the story, and/or different points in the day. These games are less linear than regular AVNs and give the players the freedom to explore the game world at their own pace.
```

Comment:
Let's say "A genre that centers its gameplay". And also "There are often story arcs", since I'm not sure that's _always_ the case. 

Also instead of "freedom", lets say "ability", since "freedom" slightly indicates positive IMO and that freedom is not necessarily always good since some people hate sandbox.